blackbox/docs/release_notes/1.2.x.txt

```diff
@@ -0,0 +1,134 @@
+.. _1.2.x_series:
+
+====================
+1.2.x Release Series
+====================
+
+Versions
+========
+
+.. toctree::
+    :maxdepth: 1
+
+    1.2.0
+
+Upgrade Guide
+=============
+
+.. NOTE::
+
+   To upgrade to the the 1.2.x release series, you must first be running the
+   latest version of the :ref:`1.1.x_series`.
+
+Daemon User
+-----------
+
+You can no longer run CrateDB as the superuser on Unix-like systems. You should
+create a new ``crate`` user for running the CrateDB daemon.
+
+
+Logging
+-------
+
+Your logging config file must be renamed::
+
+    sh$ mv logging.yml log4j2.properties
+
+System Properties
+-----------------
+
+If you have previously set or configured ``CRATE_JAVA_OPTS`` in your startup
+scripts or environment, you should remove this, as CrateDB can no longer be
+configured via this method.
+
+Configuration Changes
+---------------------
+
+Many configuration settings and files have been renamed or removed. You must
+review the release notes for :ref:`version_1.2.0` and update your setup as
+necessary.
+
+SQL Changes
+-----------
+
+Several breaking changes were made to CrateDB's SQL. This includes changes to
+time parsing, syntax changes, and new reserved keywords. You must review the
+release notes for :ref:`version_1.2.0` and update your client code as necessary.
+
+
+Bind Address
+------------
+
+The default bind address has been changed from `0.0.0.0` to the loopback
+address (meaning it will only be accessible on ``localhost``).
+
+If you bind to a network reachable IP address (e.g. you're configuring your
+cluster in a production or staging environment and not just for development
+purposes on your local machine) you must add the following line to your
+:ref:`configuration` file::
+
+    network.host: 0.0.0.0
+
+.. NOTE::
+
+   If you bind to a network reachable IP address, you must follow the
+   instructions in the next section.
+
+Elasticsearch Bootstrap Checks
```

Comment:
as far as I know, these are checks *run by* Elasticsearch, not by CrateDB. but I do not think we actually document them in our docs anywhere

perhaps we should?